src/pages/rooms/chat-list-item.tsx

```diff
@@ -75,24 +107,7 @@ export default function ChatListItem({
       />
       <div className="description">
         <ChatLinkList chat={chat} />
-        {edit || reply ? (
-          <ChatInputForm
-            onCreate={onSend}
-            onCancel={onCancel}
-            preSetInput={reply ? '' : chat.msg}
-            setInput={setInput}
-            allowEscape={true}
-          />
-        ) : (
-          <div className="content">
-            <div
-              className="text-with-markdown"
-              dangerouslySetInnerHTML={{
-                __html: emoji.emojify(sanitizeHtml(marked(chat.msg))),
-              }}
-            />
-          </div>
-        )}
+        {createdResponse()}
```

Comment:
I'd prefer to have the jsx all in one place, just do something like this for conditional Rendering:

```jsx
{!edit && msgContent}
{(edit || reply) && inputForm}
```

just with all jsx together :v:

Reply:
if I put everything in the return statement as usual I would need to duplicate the code for the message content and the input form. Does this really improve readability?

Reply:
See my code suggestion, there is no duplication of `msgContent` or `inputForm` :wink: 
If there is really no way to have the `jsx` declaration readable at one place then another component should be created